docs/data-dictionary.md

```diff
@@ -22,96 +23,122 @@ _<sup>Quick links:
-| UnitPrice                         | Number  | Storage: ms-cm-exports         | 0.0.1+  | On-demand price per unit without commitment-based discounts. This includes negotiated discounts.                                                                                                                                                                       |
-| UsageType                         | String  | Power BI: CMExport             | 0.0.1+  | Derived. Extracted from AdditionalInfo. Used for Azure Hybrid Benefit reports.                                                                                                                                                                                         |
-| <a name="v"></a>VMName            | String  | Power BI: CMExport             | 0.0.1+  | Derived. Extracted from AdditionalInfo. Used for Azure Hybrid Benefit reports.                                                                                                                                                                                         |
-| VMProperties                      | String  | Power BI: CMExport             | 0.0.1+  | Derived. Extracted from AdditionalInfo. Used for Azure Hybrid Benefit reports.                                                                                                                                                                                         |
-| VCPUs                             | Number  | Power BI: CMExport             | 0.0.1+  | Derived. Indicates the number of virtual CPUs used by this resource. Extracted from AdditionalInfo. Used for Azure Hybrid Benefit reports.                                                                                                                             |
+| Name                                       | Type    | Source ([learn more](#source)) | Version | Description                                                                                                                                                                                                                                                            |
+| ------------------------------------------ | ------- | ------------------------------ | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <a name="a"></a>AccountName                | String  | Storage: ms-cm-exports         | 0.0.1+  | Name of the identity responsible for billing for this subscription. This is your EA enrollment account owner or MOSA account admin. Not applicable to MCA.                                                                                                             |
```

Comment:
Hide whitespace to focus on the actual changes in this file:
![image](https://user-images.githubusercontent.com/399533/224766739-9fc03c6a-d067-4d45-ba0e-1ef5e2a15a7f.png)